### PR TITLE
services: host Grafana to access Loki on app.ci

### PR DESCRIPTION
**Pull request by vrutkovs**
This hosts Grafana instance with configuration to access internal Loki, so that developers would be able to query logs from CI clusters

### CODE REVIEW

File: clusters/app.ci/grafana-loki/admin_grafana.yaml [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  name: grafana-loki-admin
+  namespace: grafana-loki
+subjects:
+- kind: Group
+  apiGroup: rbac.authorization.k8s.io
+  name: grafana-loki-admins
+roleRef:
+  kind: Role
+  apiGroup: rbac.authorization.k8s.io
+  name: admin
```
Comment (stevekuznetsov) [inline clusters/app.ci/grafana-loki/admin_grafana.yaml:26]: `view` ? You should look to change YAML in the cluster using `git`

File: clusters/app.ci/grafana-loki/admin_grafana.yaml [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    app: grafana
+  type: ClusterIP
+---
+kind: Route
+apiVersion: route.openshift.io/v1
+metadata:
+  name: grafana
+  namespace: grafana-loki
+spec:
+  host: grafana-loki.ci.openshift.org
```
Comment (stevekuznetsov) [inline clusters/app.ci/grafana-loki/admin_grafana.yaml:98]: @hongkailiu do we have a doc for how to get custom domains on app.ci for your ingress?

File: clusters/app.ci/grafana-loki/admin_grafana.yaml (hunk 1 above)
Comment (vrutkovs) [reply clusters/app.ci/grafana-loki/admin_grafana.yaml]: Right, updated too view

File: clusters/app.ci/grafana-loki/admin_grafana.yaml (hunk 2 above)
Comment (vrutkovs) [reply clusters/app.ci/grafana-loki/admin_grafana.yaml]: Reused ci-search / release-controller method

**Pull request merged**